day_two_guide/topics/proc_backing-up-project.adoc

```diff
@@ -13,12 +13,11 @@ Module included in the following assemblies:
 Creating a backup of all relevant data involves exporting all important
 information, then restoring into a new project.
 
-[NOTE]
+[IMPORTANT]
 ====
-Currently, a {product-title} project back up and restore tool is being developed
-by Red Hat. See the following bug for more information:
-
-* https://bugzilla.redhat.com/show_bug.cgi?id=1303205[bugzilla 1303205].
+The following procedure utilizes the `oc get all` command, which will not list
+PVCs and Secrets. These resources will need to be back up in addition to the
```

Comment:
> PVCs and Secrets.

Should be

> PVCs and Secrets, among others.

---

> resources will need to be back up

Should be

> resources will need to be backed up

Reply:
@wgordon17, do you just need to back up PVCs and Secrets? If not, we need a list of all the items that you need to manually back up.

Reply:
🤣

This is one of those things that unfortunately doesn't work easily. So `oc get all`, lists something along the lines of
* pods
* deployments
* deploymentconfigs
* builds
* buildconfigs
* cronjobs
* services
* routes

_Normally_, the important ones to remember outside the above list include PVCs and Secrets, but there's a huge list of _potential_ things to have to back up (100 total on my testing cluster), **and** the list can differ across clusters. To get a _full_ list of everything that may _potentially_ have to be backed up, you can run `oc api-resources --namespaced=true -o name`, which will give you the list of the objects that _potentially_ may exist in the namespace. And then you'd have to `oc get` each of those object types to see if they exist in the namespace. So that was why I opt'd to mention "*among others*", but it probably wouldn't be wrong to include the longer explanation including the `oc api-resources` bit 👍🏻

Reply:
I think that it we just say "among others" that we'll eventually get a bug asking for which ones and how and why. That's a cumbersome process, though.

Reply:
> cumbersome process

🤣 That's putting it lightly